docs/modules/SDDs/pages/0006-commodore-configuration-catalog-compilation.adoc

```diff
@@ -82,33 +82,33 @@ Component versions can be any git https://git-scm.com/docs/gitglossary#Documenta
 
 ==== Secret file generation
 
-Commodore generates the secret files and their contents according to specific rules. A Kapitan secret reference, for example `?{vaultkv:path/to/secret/thekey}`, always refers to a key named thekey in a secret named path/to/secret  in Vault's KV back-end. The address of the Vault instance and the name of the back-end are configurable:
+Commodore generates the secret files and their contents according to specific rules. A Kapitan secret reference, for example `?{vaultkv:path/to/secret/thekey}`, always refers to a key named thekey in a secret named `path/to/secret` in Vault's KV back-end. The address of the Vault instance and the name of the back-end are configurable:
 
 [source,yaml]
 --
 parameters:
   secret_management:
     vault_addr: https://vault.syn.vshn.net
-    # Name of the back-end (called mount in Kapitan)
+    # Name of the back-end (called mount in Vault)
     vault_mount: kv
 --
 
-For the secret reference mentioned above, Commodore generates a Kapitan secret file in `catalog/refs/path/to/secret/thekey` with `path/to/secret:thekey`  as the reference to the Vault secret.
+For the secret reference mentioned above, Commodore generates a Kapitan secret file in `catalog/refs/path/to/secret/thekey` with `path/to/secret:thekey` as the reference to the Vault secret.
 
-Kapitan's vaultkv  secret engine is configured in class global.common with authentication method `userpass`  and the Vault address and back-end name given in `secret_management` . The configuration in `secret_management`  defaults to https://vault.syn.vshn.net and a back-end with name "kv," but this can be overridden at any level of the configuration class hierarchy.
+Kapitan's `vaultkv` secret engine is configured in the class `global.common` with authentication method `token` and the Vault address and back-end name given in `secret_management`. The configuration in `secret_management` defaults to https://vault.syn.vshn.net and a back-end with name "kv," but this can be overridden at any level of the configuration class hierarchy. The GitOps engine on the cluster uses the Vault Kubernetes authentication (in a sidecar) to lease and renew a token which can be used by Kapitan.
```

Comment:
I can not follow how one can read this and then end up with what we have. Either refine and or make an example.

Reply:
Regarding which part? Configuring the secret engine or using Vault in the cluster?

Reply:
Regarding configuration within `global.common`.

Reply:
I tried to simplify it.